docs/manual/developer/04_style_guide.md

```diff
@@ -52,6 +52,62 @@ Noteworthy additions to these principles:
 * Should use the [merge commit method](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges)
 * Should use the GitHub Web UI to document and ensure code reviews are done correctly
 
+### Commit Messages
+
+* Limit the subject line to 50 characters
+    * This is not a hard limit
+    * Keeping subject lines at this length ensures that they
+      are readable, and forces the author to think for a moment about the most concise way
+      to explain what’s going on
+    * If you’re having a hard time summarizing, you might be committing too many changes at once
+* Separate subject from body with a blank line
+    * Not every commit requires both a subject and a body.
+      Sometimes a single line is fine, especially when the change
+      is so simple that no further context is necessary
+* Capitalize the subject line
+* Do not end the subject line with a `.` (period)
+    * Trailing punctuation is unnecessary in subject lines
+* Use the imperative mood
```

Comment:
```suggestion
* Use the imperative mood in the subject line
```
I don't think that is needed in the body

Reply:
Addressed